### ISSUE TITLE
Best validation loss ?

### ISSUE THREAD

**Issue by rajarsheem** [HKUST-KnowComp/R-Net#6]
Can you share the best train/val loss you observed ?

**Comment by wzhouad** [comment]
2.82 in dev, 1.32 in train

**Comment by rajarsheem** [comment]
both are on full train & dev data respectively?

**Comment by wzhouad** [comment]
Yes
